### ISSUE TITLE
(Description & Voicing) Updates to Scene Summary & Read-Me Button Content

### ISSUE THREAD

**Issue by terracoda** [phetsims/ratio-and-proportion#411]
While creating content for the Voicing read-me buttons, I found it could be beneficial to make similar changes to the scene summary content of the Interactive Description.

Please see red and cross-out text in the [Read-Me Buttons](https://docs.google.com/document/d/1offJbONUg8y5Aa9kHewC9iTDtNISzna4giGt7WrpXCk/edit#heading=h.vj0ds4r6fj6y) section of the design doc for details.

Here's the new text without the edits visible:
### Discover Screen (Static Summary)
p1: The Discover Screen changes as you play with it. It has a Play Area and a Control Area.
p2: The Play Area is a tall space. It contains two interactive hands, a Left Hand and a Right Hand. You can move the Left Hand and Right Hand up and down individually, or activate Both Hands to move both hands at the same time. As you move the hands up and down, you will get closer to or farther from the challenge ratio, and closer or farther from a dark green screen.
p3: Tick mark options allow you to explore the vertical space with evenly-spaced horizontal lines. 

p4: There are different challenges to explore, and the Control Area has a button to reset the sim.

### Discover Screen (Dynamic Summary)
Currently, the screen is {{darkest green}}, hands are at the challenge {{1}} ratio and {{very close to}} each other:
- Left hand {{in lower region}},
- Right hand {{in lower-middle region}}.

### Discover Screen (Hint)
- see issue #399

There are changes to the Home Screen and the Discover Screen, too. I think it is best to see the formatted changes in the design doc.

But I can add them to this issue, if you would prefer.

Also, note that you need the Greenness scale implemented for the new dynamic summary:
- see issue #400

**Comment by zepumph** [comment]
Static paragraphs have been updated. The dynamic updates will need to wait on https://github.com/phetsims/ratio-and-proportion/issues/400, like you said. Marking on hold until I get to that set of regions.

**Comment by zepumph** [comment]
"of Play Area" was removed from both screen summaries. @terracoda, can you please review. It wasn't clear if you wanted any updates to the create screen summary?

**Comment by terracoda** [comment]
Yes, that's right. The changes are consistent across both screens. 
And hopefully, you got all the changes to both screens that are noted in the design doc.

I'll review in the morning, and let you know.

**Comment by terracoda** [comment]
@zepumph, so you need to make all the same changes to the PDOM descriptions on the Create Screen as you made on the Discover Screen. Apologies for not being explicit. I thought it would just happen automatically since except for a few sentences the two summaries are identical.

In addition, please add punctuation to the bulleted list items describing the position of the left hand and right hand (on both screens). I have a feeling this punctuation is needed for good pronunciation in for the web speech in the Voicing experience. I figure it is best to add it to the PDOM, since it doesn't hurt the screen reader experience, and the punctuation is correct.

In the design document, you can see the red text, red punctuation, and crossed out text that are not yet implemented for both the Discover Screen and the Create Screen. **Note** that you got everything but the punctuation on the Discover Screen. You just need to make very similar text changes to the Create Screen.

When done, the Create Screen Summary, should look like the text below.  The **_bolded and italicized text_** are the bits that are **_different from the Discover Screen_**.
> The Create Screen changes as you play with it. It has a Play Area and a Control Area.
  
> **_Here, you can create your own Challenge Ratio._** The Play Area is a tall space. It contains two interactive hands, a Left Hand and a Right Hand. You can move the Left Hand and the Right Hand up and down individually, or activate Both Hands to move both hands at the same time. As you move the hands up and down, you will get closer to or farther from the challenge ratio, unless you lock the ratio when screen is darkest green.

> Tick mark options allow you to explore the vertical space with evenly-spaced horizontal lines **_, and you can increase the range of tick marks to explore larger numbers_**.
  
> The Control Area has a button to reset the sim.

**Comment by terracoda** [comment]
Note, there is no space after "horizontal lines". The space is just there because of the formatting.

**Comment by zepumph** [comment]
All has been done, but over in https://github.com/phetsims/ratio-and-proportion/issues/400#issuecomment-974294001. I'm going to close this issue for clarity, and there can be a single issue to review these changes over there.
